src/auth/auth.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { Body, Controller, Get, Post, Req, UseGuards } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { AuthDto } from './dto/auth.dto';
+import { AuthGuard } from '@nestjs/passport';
+import { Request } from 'express';
+
+@Controller('api')
+export class AuthController {
+  constructor(private authService: AuthService) {}
+
+  @Post('users') // Đăng ký
```

Comment:
```suggestion
  @Post('users')
```
xóa comment code em nhé.
chỉ dùng khi thật cần thiết